packages/xdl/src/modules/Modules.js

```diff
@@ -39,19 +44,23 @@ export function getVersionableModulesForPlatform(platform: Platform): Array<Modu
   });
 }
 
-export function getDetachableModulesForPlatformAndSdkVersion(
-  platform: Platform,
-  sdkVersion: string
-): Array<ModuleConfig> {
-  return getAllForPlatform(platform).filter(moduleConfig => {
-    return (
-      moduleConfig.isNativeModule &&
-      moduleConfig.detachable &&
-      (sdkVersion === 'UNVERSIONED' || semver.satisfies(sdkVersion, moduleConfig.sdkVersions))
-    );
-  });
+export function getDetachableModules(platform: Platform, sdkVersion: string): Array<ModuleConfig> {
+  return getAllForPlatform(platform, sdkVersion).filter(
+    moduleConfig => moduleConfig.isNativeModule && moduleConfig.detachable
+  );
+}
+
+export function getPublishableModules(sdkVersion: string): Array<ModuleConfig> {
+  return expoSdkUniversalModulesConfigs.filter(
+    moduleConfig =>
+      !!moduleConfig.libName && doesVersionSatisfy(sdkVersion, moduleConfig.sdkVersions)
+  );
 }
 
-export function getPublishableModules(): Array<ModuleConfig> {
-  return expoSdkUniversalModulesConfigs.filter(moduleConfig => !!moduleConfig.libName);
+function doesVersionSatisfy(version: string, versionRequirement: string): boolean {
+  if (version === 'UNVERSIONED') {
+    return semver.satisfies(HIGHEST_KNOWN_VERSION, versionRequirement);
```

Comment:
where is `HIGHEST_KNOWN_VERSION` defined?

Reply:
whoops, I removed it when refactoring 😅

what would you say to removing handling `UNVERSIONED` altogether?

Reply:
iirc this was required for sth to work, but I don't remember for what 😅 maybe you can try 😂

Reply:
ah, for building ios shell apps, nvm